diff --git src/port/pg_bitutils.c src/port/pg_bitutils.c
index 7847e8a..17714af 100644
--- src/port/pg_bitutils.c
+++ src/port/pg_bitutils.c
@@ -12,12 +12,14 @@
  */
 #include "c.h"

+#if !defined(__ppc__) && !defined(__PPC__) && !defined(__ppc64__) && !defined(__PPC64__)
 #ifdef HAVE__GET_CPUID
 #include <cpuid.h>
 #endif
 #ifdef HAVE__CPUID
 #include <intrin.h>
 #endif
+#endif

 #include "port/pg_bitutils.h"

@@ -141,6 +143,7 @@ int			(*pg_popcount64) (uint64 word) = pg_popcount64_slow;
 static bool
 pg_popcount_available(void)
 {
+#if !defined(__ppc__) && !defined(__PPC__) && !defined(__ppc64__) && !defined(__PPC64__)
 	unsigned int exx[4] = {0, 0, 0, 0};

 #if defined(HAVE__GET_CPUID)
@@ -152,6 +155,9 @@ pg_popcount_available(void)
 #endif

 	return (exx[2] & (1 << 23)) != 0;	/* POPCNT */
+#else
+	return false;
+#endif
 }

 /*
